src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/InbreedingCoeff.java

```diff
@@ -69,16 +69,16 @@ public InbreedingCoeff(final Set<String> founderIds){
             logger.warn("Annotation will not be calculated, must provide at least " + MIN_SAMPLES + " samples");
             return Collections.emptyMap();
         }
-        return Collections.singletonMap(getKeyNames().get(0), (Object) String.format("%.4f", F));
+        return Collections.singletonMap(getKeyNames().get(0), String.format("%.4f", F));
     }
 
     @VisibleForTesting
-    Pair<Integer, Double> calculateIC(final VariantContext vc, final GenotypesContext genotypes) {
-        final GenotypeCounts t = GenotypeUtils.computeDiploidGenotypeCounts(vc, genotypes);
+    static Pair<Integer, Double> calculateIC(final VariantContext vc, final GenotypesContext genotypes) {
+        final GenotypeCounts t = GenotypeUtils.computeDiploidGenotypeCounts(vc, genotypes, false);
```

Comment:
Extract a `static final boolean ` constant `ROUND_GENOTYPE_COUNTS` equal to `false`, and use it here for clarity.